Happy Dance/Assets/_Scripts/Speech.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+
+using UnityEngine;
+
+/// <summary>
+/// Speech is used to store the audioclip that will be played and the string that will be displayed when the system wants to talk to the player
+/// </summary>
+[Serializable]
+public class Speech
+{
+    public AudioClip SpeechAudioClip;
+    [TextArea] public string SpeechText;
```

Comment:
I need to make a policy on this, but I think that [Attributes] should be on a line by themselves.